src/meta/src/manager/streaming_job.rs

```diff
@@ -16,18 +16,37 @@ use std::collections::HashMap;
 
 use risingwave_common::catalog::TableVersionId;
 use risingwave_common::util::epoch::Epoch;
-use risingwave_pb::catalog::{Index, Sink, Source, Table};
+use risingwave_pb::catalog::{Index, PbSource, Sink, Table};
+use risingwave_pb::ddl_service::PbTableJobType;
 
 use crate::model::FragmentId;
 
+#[derive(Default, Debug, Clone)]
+pub enum TableJobType {
```

Comment:
Why not just `boolean`? 🤔

Reply:
I think enum is more readable, and is more extensible to new scenarios.

Reply:
Maybe we can directly use the protobuf enum. There seems to be no difference.